lammps_plugin/README.rst

```diff
@@ -70,12 +70,12 @@ As an example for a multicomponet system containing elements 'A' and 'B' the abo
    pair_coeff 1 1 A_A
    pair_coeff 1 2 A_B
    pair_coeff 2 2 B_B
-   pair_coeff 1 1 1 A_A_A
-   pair_coeff 1 1 2 A_A_B
-   pair_coeff 1 2 2 A_B_B
-   pair_coeff 2 1 1 B_A_A
-   pair_coeff 2 1 2 B_A_B
-   pair_coeff 2 2 2 B_B_B
+   pair_coeff 1 1 1 1 1 A_A_A
```

Comment:
This is a confusing way to define the 3-Body inputs. If we progress with this format of the first two numbers being ignored I suggest changing the README to the following convention:

```
pair_coeff * * 1 1 1 A_A_A
pair_coeff * * 1 1 2 A_A_B
```

Reply:
In this format, it is easier to read the interactions being defined